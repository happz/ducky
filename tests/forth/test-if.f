: T CR TRUE  IF 65 66 67 ELSE 68 69 70 THEN ;
T

: T CR FALSE IF 65 66 67 ELSE 68 69 70 THEN ;
T

CR EMIT EMIT EMIT EMIT EMIT EMIT CR

