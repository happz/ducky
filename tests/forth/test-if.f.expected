Ducky Forth welcomes you




FEDCBA
